src/scdaq.cc

```diff
@@ -58,12 +58,11 @@ int run_pipeline( int nbThreads, ctrl& control, config& conf )
       TOTAL_SLICES = conf.getNumInputBuffers();
       
       //input_filter = std::make_shared<FileInputFilter>( conf.getInputFile(), MAX_BYTES_PER_INPUT_SLICE, TOTAL_SLICES );
-      throw std::runtime_error("input type FILE is temporarily not supported");
+      throw std::runtime_error("input type FILE is not supported");
```

Comment:
Hi @pzejdl,

If we don't intend to support the FILE input, shouldn't we just remove this completely including the config option?

-d

Reply:
Agree,
The intention was to carry on the source files in case it is resurrected at some point, similarly to DMA Input Filter. 

However, in this case you are right. We have file DMA reader that replaced this file input, so it can be removed.

Reply:
Being solved in a separate PR: #26